src/modules/users/components/TokenActivation/TokenActivationContent/ChangeTokenStateForm.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useCallback, useState } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { BigNumber, bigNumberify } from 'ethers/utils';
+import { FormikProps } from 'formik';
+import moveDecimal from 'move-decimal-point';
+import * as yup from 'yup';
+
+import Button from '~core/Button';
+import { ActionForm, Input } from '~core/Fields';
+import Numeral from '~core/Numeral';
+
+import { UserToken } from '~data/generated';
+import { ActionTypes } from '~redux/index';
+import { pipe, mapPayload } from '~utils/actions';
+
+import styles from './TokenActivationContent.css';
+
+const MSG = defineMessages({
+  changeState: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.changeState',
+    defaultMessage: 'Change token state',
+  },
+  activate: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.activate',
+    defaultMessage: 'Activate',
+  },
+  withdraw: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.withdraw',
+    defaultMessage: 'Withdraw',
+  },
+  balance: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.balance',
+    defaultMessage: 'balance: {tokenBalance}',
+  },
+});
+
+const validationSchema = yup.object({
+  amount: yup.number().required().moreThan(0),
+});
+
+type FormValues = {
+  amount: number;
+};
+
+export interface ChangeTokenStateFormProps {
+  token: UserToken;
+  tokenDecimals: number;
+  activeTokens: BigNumber;
+  inactiveTokens: BigNumber;
+}
+
+const ChangeTokenStateForm = ({
+  token,
+  tokenDecimals,
+  activeTokens,
+  inactiveTokens,
+}: ChangeTokenStateFormProps) => {
+  const [isActivate, setIsActivate] = useState(true);
+
+  const formAction = useCallback(
+    (actionType: '' | '_ERROR' | '_SUCCESS') =>
```

Comment:
Might not need to use the underscore `_` in the name, but you call :)

However please don't use a empty string as a value or a type. Give it some name, doesn't really matter is it's only being used locally.